Startup.cs

```diff
@@ -0,0 +1,46 @@
+﻿using EcommerceStore.Data.Context;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+
+namespace EcommerceStore
+{
+    public class Startup
+    {
+        public Startup(IConfiguration configuration)
+        {
+            Configuration = configuration;
+        }
+
+        public IConfiguration Configuration { get; }
+
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddDbContext<EcommerceContext>(opt => opt.UseNpgsql(Configuration.GetConnectionString("EcommerceConnection")));
+
+            services.AddControllers();
```

Comment:
you can turn of the default [ApiController] attribute behaviour https://stackoverflow.com/questions/60100437/model-state-validation-in-web-api
Let's discuss offline when this can be useful